khmer/_oxli/graphs.pyx

```diff
@@ -391,31 +391,43 @@ cdef class QFCounttable(Hashtable):
 
 cdef class Counttable(Hashtable):
 
-    def __cinit__(self, int k, uint64_t starting_size, int n_tables):
-        cdef vector[uint64_t] primes
+    def __cinit__(self, int k, uint64_t starting_size, int n_tables,
+                  primes=[]):
```

Comment:
I'm not sure how Cython treats this, but in Python defining arguments as lists or dicts is quite dangerous (they get allocated at definition time, and not in every function call). I suggest setting it to `None` and doing a check in the body:
```
if primes is None:
    primes = []
```